doc/source/handling-dependencies.rst

```diff
@@ -0,0 +1,216 @@
+.. _handling_dependencies:
+
+Handling Dependencies
+=====================
+
+Your Ray application may depend on environment variables, files, and Python packages.
+Ray provides two features to specify these dependencies when working with a remote cluster: Runtime environments, and the Ray cluster launcher commands
+With these features, you no longer need to manually SSH into your cluster and set up your environment.
```

Comment:
A little confused with this sentence. We don't need to manually SSH into your cluster for the existing solution now right? (it is handled by the setup commands)

Reply:
Yeah, I meant to include setup commands in "Ray cluster launcher commands", which this doc describes as an existing feature.  Let me make this more clear